#5 feat(django app): 유저 관리를 위한 새로운 Django 앱 생성 및 등록

uridonget opened the pull request:
# 유저관리를 위한 새로운 앱 등록 \#5 
## Overview
<!---- 변경 사항 및 관련 이슈에 대해 간단하게 작성해주세요. 어떻게보다 무엇을 왜 수정했는지 설명해주세요. -->
유저관리를 용이하게 할 수 있도록 새로운 account 장고앱을 생성.
장고의 설정파일(settings.py)에 새로운 앱을 등록
엔드포인트 관리를 용이하게 할 수 있도록 urls.py 파일을 account 디렉토리에 생성.
프로젝트 디렉토리의 urls.py에 account.urls를 추가하여 엔드포인트를 통합하여 관리.
새로운 account 앱의 views.py에 테스트 함수를 추가하여 의도대로 잘 작동하는지 확인.

<!---- Resolves: #(Isuue Number) -->
## PR Type


- [x] feat (새로운 앱 등록)
- [x] tests (/account/ 으로 접속)

## PR Checklist
PR이 다음 요구 사항을 충족하는지 확인하세요.

- [x] 커밋 메시지 컨벤션에 맞게 작성했습니다.
- [x] 변경 사항에 대한 테스트를 했습니다.(버그 수정/기능에 대한 테스트).

uridonget committed:
#5 feat(django app): 유저 관리를 위한 새로운 Django 앱 생성 및 등록

python manage.py startapp account 명령어를 사용하여 새로운 account 앱을 생성.
프로젝트 디렉토리의 settings.py의 INSTALLED_APP에 새로 생성된 'account' 추가.
앱 단위로 url 엔드포인트 관리를 용이하게 할 수 있도록 account 디렉토리 내부에 urls.py 생성.
프로젝트 디렉토리의 urls.py에 include 기능을 통해 account 앱에서 설정된 url을 통합하여 관리.
테스트를 위해 views.py에 간단한 함수를 추가. [엔드포인트: /account/].

eldeshue commented:
formatter 체크에서 실패가 있네요! formatter를 돌리고 push합시다!

uridonget committed:
#5 fix(django app): black 포맷에 맞추어 수정 #6

django 컨테이너에 설치하는 라이브러리 목록인 requirements.txt 파일에 black을 추가.
새로 생성된 account 디렉토리에 black formatter를 실행.

uridonget commented:
black formatter 돌리고 다시 push 했습니다!
lint 스타일 지적 사항은 잘 모르겠네요. 가르침과 피드백 부탁드립니다..!

eldeshue commented on backend/account/views.py:
```diff
@@ -0,0 +1,7 @@
+# Create your views here.
+
+def test(request):
+    return HttpResponse("Hello, World!\n")
```
테스트 관련 코드를 실재 구현이 들어갈 파일에 작성하는 것은 적절하지 않다고 생각합니다. 👍

uridonget replied:
동의합니다! 해당 파일의 테스트 관련 함수를 삭제하고 해당 함수를 호출하는 urls.py의 엔드포인트도 삭제하였습니다! ❤️

uridonget committed:
#5 fix(django app): 불필요한 테스트 코드 삭제 #6

account 디렉토리의 views.py 파일에 작성된 불필요한 test 함수를 삭제.
해당 함수를 호출하는 urls.py의 endpoint 또한 삭제.

eldeshue reviewed (approved):
test 관련 코드 삭제 확인했습니다. LGTM!
